src/core/tokens/_core-typography.scss

```diff
@@ -0,0 +1,50 @@
+// CORE TOKENS: TYPOGRAPHY
+// This partial Defines Global Typography tokens
+
+// FONTS SCALES
+// - Font Size scale
+$font-size-10: $font-reg-size-14;
```

Comment:
why are we creating these scale variables?

Reply:
this is the case right now.
i did not change it, just wrote it in a token that is generate from a map using a mixin.
the output is the same.

And i agree , this should not be exposed.
Yet in my scope i did not change the output besides removing duplication and changing sort order in some cases.